operators/README.md

```diff
@@ -55,6 +55,8 @@ helm upgrade crownlabs-instance-operator deploy/instance-operator \
 
 Based on [Kubebuilder 2.3](https://github.com/kubernetes-sigs/kubebuilder.git), the operator implements the environment creation logic of CrownLabs.
 
+![Instance Operator Architecture](../documentation/instance-operator.jpg)
```

Comment:
Three comments here:
1. From my point of view, the image should be accompanied by some descriptive text to contextualize what the figure represents and which are the main elements to look at. Otherwise it may be difficult for a newbie reader to understand many concepts. 
2. There are some inaccuracies. For instance, the VM is also created by the operator, the oauth2-proxy components are missing, and it is not clear to me the meaning of the arrow going from the secret/service/ingress to the VMI.
3. It would be better if the picture were in the svg format, as it scales better and it prevents the issues typical of raster graphics. Additionally, it could be useful to check-in also the source file, to allow for easy modifications if the workflow is modified (e.g. docker-based instances).